src/tests/ilverify/ILTests/InitOnlyTest.il

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This is an explicit test for #55033, we can delete it after the CI run, as these scenarios are covered by FieldTests.il too.

Reply:
The `IsExternalInit` case is not covered by FieldTests.il. I would just add the new cases to FieldTests.il if they are not covered there. For test execution efficiency, it is better to have fewer larger tests projects.

Reply:
Original PR was about ldflda and later expanded to cover stfld/IsExternalInit.  It was more of a reminder to self which I was going to address anyway. :)

> For test execution efficiency, it is better to have fewer larger tests projects.

FWIW, ILVerification.Tests.csproj is not a large and anyway priority1 project, so it only executes on subset of CI runs. Its build doesn't take more than a few/three seconds, and `ILVerification.Tests.sh` (that runs all ilverification tests) completed in 0.96s on my macOS.